Building V in MSYS fails

### Describe the bug

```
$ make
make fresh_vc
make[1]: Entering directory '/c/repos/v'
rm -rf ./vc
git clone --filter=blob:none --quiet https://github.com/vlang/vc ./vc
remote: Enumerating objects: 3, done.
remote: Counting objects: 100% (3/3), done.
remote: Compressing objects: 100% (3/3), done.
remote: Total 3 (delta 1), reused 0 (delta 0), pack-reused 0 (from 0)
Receiving objects: 100% (3/3), 1.09 MiB | 2.56 MiB/s, done.
Resolving deltas: 100% (1/1), done.
make[1]: Leaving directory '/c/repos/v'
cd ./vc && git clean -xf && git pull --quiet
make fresh_tcc
make[1]: Entering directory '/c/repos/v'
rm -rf ./thirdparty/tcc
git clone --filter=blob:none --quiet --branch thirdparty-windows-amd64 https://github.com/vlang/tccbin ./thirdparty/tcc
remote: Enumerating objects: 165, done.
remote: Counting objects: 100% (20/20), done.
remote: Compressing objects: 100% (20/20), done.
remote: Total 165 (delta 1), reused 8 (delta 0), pack-reused 145 (from 1)
Receiving objects: 100% (165/165), 988.93 KiB | 1.22 MiB/s, done.
Resolving deltas: 100% (23/23), done.
make[1]: Leaving directory '/c/repos/v'
cd ./thirdparty/tcc && git clean -xf && git pull --quiet
cc  -std=c99 -municode -w -o v1.exe ./vc/v_win.c  -lws2_32
v1.exe -no-parallel -o v2.exe  cmd/v
make: v1.exe: No such file or directory
make: *** [GNUmakefile:102: all] Error 127
```

### Reproduction Steps

1. Download msys from official website
2. Open its shell
3. git clone git@github.com:vlang/v.git
4. cd v
5. make

### Expected Behavior

successful build

### Current Behavior

build fails

### Possible Solution

Add './' before executables

### Additional Information/Context

_No response_

### V version

V 0.4.8 8ffcc34

### Environment details (OS name and version, etc.)

windows 10
MINGW64_NT-10.0-19045 DESKTOP-GM3H7ET 3.5.4-0bc1222b.x86_64 2024-09-04 18:28 UTC x86_64 Msys

<sub><a href="https://huly.app/guest/vlang-66f40c4d-a476b54c67-771fdd?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJsaW5rSWQiOiI2NzFlOWUzMDI3Y2M3NjgxYzYyMjJjNGEiLCJndWVzdCI6InRydWUiLCJlbWFpbCI6IiNndWVzdEBoYy5lbmdpbmVlcmluZyIsIndvcmtzcGFjZSI6InctYWxleGFuZGVyLXZsYW5nLTY2ZjQwYzRkLWE0NzZiNTRjNjctNzcxZmRjIn0.YUYurmVU61oC_D-gD_hFyA6N9QWWYuf8oR60QYfVIi0">Huly&reg;: <b>V_0.6-21122</b></a></sub>
> [!NOTE]
> You can use the 👍 reaction to increase the issue's priority for developers.
>
> Please note that only the 👍 reaction to the issue itself counts as a vote.
> Other reactions and those to comments will not be taken into account.

--- a/GNUmakefile
+++ b/GNUmakefile
@@ -99,8 +99,8 @@ endif
 all: latest_vc latest_tcc latest_legacy
 ifdef WIN32
 	$(CC) $(CFLAGS) -std=c99 -municode -w -o v1.exe $(VC)/$(VCFILE) $(LDFLAGS) -lws2_32
-	v1.exe -no-parallel -o v2.exe $(VFLAGS) cmd/v
-	v2.exe -o $(VEXE) $(VFLAGS) cmd/v
+	./v1.exe -no-parallel -o v2.exe $(VFLAGS) cmd/v
+	./v2.exe -o $(VEXE) $(VFLAGS) cmd/v
 	$(RM) v1.exe
 	$(RM) v2.exe
 else

